e2e/atlas/integrations_test.go

```diff
@@ -0,0 +1,307 @@
+			"--flowName",
+			"test",
+			"--orgName",
+			"testOrg",
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		a := assert.New(t)
+		a.NoError(err, string(resp))
+
+		var thirdPartyIntegrations mongodbatlas.ThirdPartyIntegrations
+		if err := json.Unmarshal(resp, &thirdPartyIntegrations); a.NoError(err) {
+			found := false
+			services := thirdPartyIntegrations.Results
+			for i := range services {
+				if services[i].Type == flowdockEntity {
+					found = true
+					break
+				}
+			}
+			a.True(found)
+		}
+	})
+
+	t.Run("Create NEW_RELIC", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			integrationsEntity,
+			"create",
+			newRelicEntity,
+			"--accountId",
+			key,
+			"--licenceKey",
+			key,
+			"--writeToken",
+			key,
+			"--readToken",
+			key,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		a := assert.New(t)
+		a.NoError(err, string(resp))
+
+		var thirdPartyIntegrations mongodbatlas.ThirdPartyIntegrations
+		if err := json.Unmarshal(resp, &thirdPartyIntegrations); a.NoError(err) {
+			found := false
+			services := thirdPartyIntegrations.Results
+			for i := range services {
+				if services[i].Type == newRelicEntity {
+					found = true
+					break
+				}
+			}
+			a.True(found)
+		}
+	})
+
+	t.Run("Create OPSGENIE", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			integrationsEntity,
+			"create",
+			opsGenieEntity,
+			"--apiKey",
+			key,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		a := assert.New(t)
+		a.NoError(err, string(resp))
+
+		var thirdPartyIntegrations mongodbatlas.ThirdPartyIntegrations
+		if err := json.Unmarshal(resp, &thirdPartyIntegrations); a.NoError(err) {
+			found := false
+			services := thirdPartyIntegrations.Results
+			for i := range services {
+				if services[i].Type == opsGenieEntity {
+					found = true
+					break
+				}
+			}
+			a.True(found)
+		}
+	})
+
+	t.Run("Create PAGER_DUTY", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			integrationsEntity,
+			"create",
+			pagerDutyEntity,
+			"--serviceKey",
+			key,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		a := assert.New(t)
+		a.NoError(err, string(resp))
+
+		var thirdPartyIntegrations mongodbatlas.ThirdPartyIntegrations
+		if err := json.Unmarshal(resp, &thirdPartyIntegrations); a.NoError(err) {
+			found := false
+			services := thirdPartyIntegrations.Results
+			for i := range services {
+				if services[i].Type == pagerDutyEntity {
+					found = true
+					break
+				}
+			}
+			a.True(found)
+		}
+	})
+
+	t.Run("Create VICTOR_OPS", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			integrationsEntity,
+			"create",
+			victorOpsEntity,
+			"--apiKey",
+			key,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		a := assert.New(t)
+		a.NoError(err, string(resp))
+	})
```

Comment:
VICTOR_OPS is not inside the returned list. I have created this task to investigate this behaviour [CLOUDP-71439](https://jira.mongodb.org/browse/CLOUDP-71439)